web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
I think you won't need all that. Just to clarify the 4 levels:

1. app drawer icon
2. top bar menu item
3. item when a top bar menu item is a dropdown
4. item below 3 when 3 is `.dropdown-header`

\# 1-3 are covered, and # 4 should be, by dropdown-header-ing # 3.